receiver/vcenterreceiver/scraper_test.go

```diff
@@ -32,14 +33,37 @@ import (
 
 func TestScrape(t *testing.T) {
 	ctx := context.Background()
-	mockServer := mock.MockServer(t)
+	mockServer := mock.MockServer(t, false)
 
 	cfg := &Config{
 		Metrics:  metadata.DefaultMetricsSettings(),
 		Endpoint: mockServer.URL,
 		Username: mock.MockUsername,
 		Password: mock.MockPassword,
 	}
+
+	testScrape(t, ctx, cfg)
```

Comment:
```suggestion
	testScrape(ctx, t, cfg)
```

Fixing ordering due to lint, see later comment.